### PR TITLE
Replace the dapis section with the dapps section

### PR DESCRIPTION
**Pull request by bbenligiray**
Market/integration portion of https://github.com/api3dao/oev-docs/issues/97

To do in another PRs:
- Security considerations (in the "Integration" collapsible)
- An OEV Rewards collapsible
  - Manual payments + a coming soon section for API3 DEAL
  - Some explanation about how to get a dApp ID assigned + possibly a dApp ID list that will temporarily live here

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -9,8 +9,7 @@
     {
       "files": "*.md",
       "options": {
-        "parser": "markdown",
-        "proseWrap": "always"
```
Comment (bbenligiray) [inline .prettierrc:13]: pls

File: docs/index.md [hunk 2]
```diff
@@ -3,17 +3,15 @@ layout: home
...
+  text: Oracles that pay you
+  tagline: Receive OEV Rewards for your DeFi project
   image:
     src: /img/beacons.svg
     alt: API3
   actions:
     - theme: brand
-      text: dAPIs
-      link: /dapis/
+      text: dApps
```
Comment (bbenligiray) [inline docs/index.md:13]: I think this should `For dApps`/`For OEV searchers` rather than `dApps`/`OEV searchers`

**Comment by bbenligiray** [comment]
I'm not sure why the link checker fails

**Review by Siegrift — APPROVED** [review]
> I'm not sure why the link checker fails

You should not use relative paths. E.g. instead of `[contract integration page](./contract-integration.md)` use `[contract integration page](/dapps/integration/contract-integration)`. And one of the broken links no longer exists.

I've read all of the content, but haven't tested the Quickstart myself.

**Pull request merged**